src/mapping/PetRadialBasisFctMapping.hpp

```diff
@@ -590,10 +590,19 @@ void PetRadialBasisFctMapping<RADIAL_BASIS_FUNCTION_T>::computeMapping()
     auto                  rescalingCoeffs = petsc::Vector::allocate(_matrixC);
     VecSet(rhs, 1);
     rhs.assemble();
-    if (not _solver.solve(rhs, rescalingCoeffs)) {
-      PRECICE_WARN("RBF rescaling linear system has not converged. Deactivating rescaling!");
+    switch (_solver.solve(rhs, rescalingCoeffs)) {
+    case (petsc::KSPSolver::SolverResult::Converged):
+      PRECICE_INFO("Using rescaling. " << _solver.summaryFor(rhs));
+      break;
+    case (petsc::KSPSolver::SolverResult::Stopped):
+      PRECICE_WARN("Using rescaling, but beware. " << _solver.summaryFor(rhs));
+      break;
```

Comment:
We should also deactivate rescaling in this case. Will be useless I guess.

Reply:
In this case, the solver didn't reach the required relative convergence.
The result should still be usable though, right?

Reply:
Not sure. For this, I don't yet understand the rescaling too well :grin: . But I think it could be dangerous. And switching off does not hurt. Most probably, the actual mapping will then not converge.
Problematic could be if the rescaling runs into this false convergence and the mapping later into a real convergence. Then the user might think that she can fully trust the mapping. Which she maybe cannot.